CMakeLists.txt

```diff
@@ -58,6 +58,13 @@ function(packageProject)
     COMPATIBILITY ${PROJECT_COMPATIBILITY}
   )
 
+  message(
+    STATUS "LIBRARY DESTINATION=${CMAKE_INSTALL_LIBDIR}/${PROJECT_NAME}${PROJECT_VERSION_SUFFIX}"
+  )
+  message(
+    STATUS "RUNTIME DESTINATION=${CMAKE_INSTALL_BINDIR}/${PROJECT_NAME}${PROJECT_VERSION_SUFFIX}"
+  )
+  message(STATUS "PUBLIC_HEADER DESTINATION=${PROJECT_INCLUDE_DESTINATION}")
```

Comment:
TBH I don't like all the outputs as it's only really relevant for debugging PackageProject itself. Also note that this would be printed for each dependency using PackageProject, resulting in a very verbose configuration.

Reply:
Yes, for one project, it may informative, but for many, it becomes noice.